package/CHANGELOG

```diff
@@ -19,6 +19,8 @@ The rules for this file:
   * 2.0.0
 
 Fixes
+  * Instead of using ATOM for both ATOM and HETATM, HETATM record type
```

Comment:
There's an author list for version 2.0.0 above (line 17), please do add yourself there too :)

Reply:
Done